tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -362,19 +363,19 @@ Check Versions In JupyterLab
         @{libDetail} =  Split String  ${libString}  ${SPACE}v
         IF  "${libDetail}[0]" == "TensorFlow"
             ${status} =  Verify Installed Library Version  tensorflow-gpu  ${libDetail}[1]
-            IF  '${status}' == 'FAIL'
+            IF  '${status}[0]' == 'FAIL'
```

Comment:
I think it would be cleaner to do 
```
${status}  ${value}  =  Verify Installed Library Version  tensorflow-gpu  ${libDetail}[1]
IF  '${status}' == 'FAIL'
```

Same for the other two

Reply:
Understood, I will do it

Reply:
Done